src/univariate/continuous/normal.jl

```diff
@@ -74,22 +73,21 @@ kurtosis(d::Normal{T}) where {T<:Real} = zero(T)
 
 entropy(d::Normal) = (log2π + 1)/2 + log(d.σ)
 
-
 #### Evaluation
 
-@_delegate_statsfuns Normal norm μ σ
-
-gradlogpdf(d::Normal, x::Real) = (d.μ - x) / d.σ^2
-
-mgf(d::Normal, t::Real) = exp(t * d.μ + d.σ^2/2 * t^2)
-cf(d::Normal, t::Real) = exp(im * t * d.μ - d.σ^2/2 * t^2)
-
+logpdf(d::Normal, x::Real) = -(x - d.μ)^2 / 2 / d.σ^2 - log(2π * d.σ^2) / 2
+pdf(d::Normal, x::Real) = exp(logpdf(d, x))
+cdf(d::Normal, x::Real) = (1 + erf((x - d.μ) / d.σ / √2)) / 2
+function logcdf(d::Normal, x::Real)
+    value = erf((x - d.μ) / d.σ / √2)
+    value < -1 ? log1p(value) - log(2) : log((1 + value) / 2)
```

Comment:
wait two possible computations for logcdf of a normal. Do you have a reference paper/doc on this?

Reply:
Same formula. The only difference is using the `log` or `log1p` according to the docstring of `log1p`. 
```
log1p(x)

  Accurate natural logarithm of 1+x. Throws DomainError for Real arguments
  less than -1.
```

Reply:
ok, try to write it in a consistent way (either -log(2) or the /2 in the log function. Add a comment regarding the use of `log1p` and why not use it in all cases? You are here using it only when it will throw an error
Also, use `ifelse` instead of the ternary operation, usually a tad faster for small branches like this.

Reply:
Actually, the codomain of the error function (`erf`) is [-1, 1], so it would be safe to always use `log1p`. Is there any preference in terms of numerical precision on whether it would be best to divide and take the log afterwards or add the log of the factorization?

Reply:
no idea but I would think not

Reply:
if you want to be picky, log2 can be a pre-computed constant

Reply:
I will use `StatsFuns.logtwo`